internal/languages/golang/detectors/.snapshots/TestGoObjects-object_no_class

```diff
@@ -109,8 +109,6 @@ children:
                       id: 22
                       range: 6:2 - 6:6
                       content: user
-                      alias_of:
```

Comment:
Not sure about the snapshot changes - @didroe any concerns?

Reply:
I think this is ok, let's see if all the rules tests pass once it's merged